src/api/services/usersService.ts

```diff
@@ -71,5 +96,9 @@ export const createUserInPortal = async (
 };
 
 export const getAllUserFromParticipant = async (participant: Participant) => {
```

Comment:
This is the fix for duplicate team members from showing on the team members page if they have more than one userRoleId for that participant